lxc/file.go

```diff
@@ -824,7 +846,7 @@ func (c *cmdFilePush) run(cmd *cobra.Command, args []string) error {
 	defer reverter.Fail()
 
 	// Make sure all of the files are accessible by us before trying to push any of them
-	var files []*os.File
+	files := make([]*os.File, len(sourcefilenames))
```

Comment:
```suggestion
	files := make([]*os.File, 0, len(sourcefilenames))
```
Since we are appending to it, better have the initial length set to 0, I think.